packages/openai-adapter/tests/Unit/Chat/OpenaiChatAdapterTest.php

```
@@ -559,6 +561,71 @@ public function testSupportResponseFormatWithUnsupportedInstance(): void
+            fn () => null,
+        );
+
+        $adapter = new OpenaiChatAdapter($client);
+        $result = $adapter->handleRequest($request);
+
+        $this->assertInstanceOf(AIChatResponseStream::class, $result);
+
+        // Register callback to track usage updates
+        $receivedUpdates = [];
+        $result->registerUsageCallback(new class($receivedUpdates) implements UsageCallbackInterface {
+            public function __construct(private array &$updates)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$updates'. (undefined)

(UnusedFormalParameter)